packages/augur-ui/src/modules/market-charts/components/depth/depth.tsx

```diff
@@ -3,11 +3,12 @@ import * as d3 from 'd3';
 import ReactFauxDOM from 'react-faux-dom';
 import memoize from 'memoizee';
 import { createBigNumber } from 'utils/create-big-number';
-import { ASKS, BIDS, BUY, SELL, ZERO } from 'modules/common/constants';
+import { ASKS, BIDS, BUY, SELL, ZERO, THEMES } from 'modules/common/constants';
```

Comment:
Revert all the changes to depth.tsx, this ticket is for styling only and shouldn't touch this file. I know this probably needs changes but it's going to be a separate ticket because it will require re-drawing on theme switch among other things.